Core/include/Acts/Propagator/MultiEigenStepperLoop.hpp

```diff
@@ -759,6 +759,21 @@ class MultiEigenStepperLoop
       const FreeToBoundCorrection& freeToBoundCorrection =
           FreeToBoundCorrection(false)) const;
 
+  /// @brief If necessary fill additional members needed for curvilinearState
+  ///
+  /// Compute path length derivatives in case they have not been computed
+  /// yet, which is the case if no step has been executed yet.
+  ///
+  /// @param [in, out] prop_state State that will be presented as @c BoundState
+  /// @param [in] navigator the navigator of the propagation
+  /// @return true if nothing is missing after this call, false otherwise.
+  template <typename propagator_state_t, typename navigator_t>
+  bool prepareCurvilinearState(
```

Comment:
Does this not have the same issue? Does the correction call somehow get forwarded to the `EigenStepper`, and I'm not seeing it?